tfjs-backend-webgpu/src/setup_test.ts

```diff
@@ -126,6 +126,16 @@ const TEST_FILTERS: TestFilter[] = [
       'fused',     // Not yet implemented.
     ]
   },
+  {
+    include: 'fused conv2d',
+    excludes: [
+      'im2row with prelu',                         // Actual != expected.
```

Comment:
Hm, how big is the error? I'm surprised that these tests needed to be excluded.

Reply:
The reported error is like this on Mac:
Arrays differ: actual[7] = 0, expected[7] = -2.5.